docs/en/enterprise-edition/policy-reference/sast-policies/go-policies/sast-policy-262.adoc

```diff
@@ -0,0 +1,85 @@
+== Use of binding to all network interfaces
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|=== 
+|Prisma Cloud Policy ID 
+| 6f0a5b73-f697-4cc8-9497-e8056dc830dc
+
+|Checkov ID 
+|CKV3_SAST_262
+
+|Severity
+|MEDIUM
+
+|Subtype
+|Build
+
+|Language
+|Go
+
+|CWEs
+|https://cwe.mitre.org/data/definitions/200.html[CWE-200: Exposure of Sensitive Information to an Unauthorized Actor]
+
+|OWASP Categories
+|https://owasp.org/Top10/A05_2021-Security_Misconfiguration/[A05:2021-Security Misconfiguration]
+
+|=== 
+
+=== Description
+
+This policy identifies instances where Go applications bind to all network interfaces using `0.0.0.0` or `[::]`. Binding to all interfaces can expose services to unintended and potentially insecure traffic. It is recommended to bind to specific interfaces by passing the interface IP address through an environment variable, configuration file, or by determining the primary interface(s) IP address.
```

Comment:
This policy checks whether Go applications bind to all network interfaces using `0.0.0.0` or `[::]`. Binding to all interfaces can expose services to unintended and potentially insecure traffic.

@tsmithv11 -moved the remediation